packages/api/api-generator/src/commands/generate/generateCrud/__tests__/__snapshots__/sort-by-id.spec.ts.snap

```diff
@@ -0,0 +1,30 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`sort by id id not be default value if sort is disabled 1`] = `
+"import { ArgsType, Field, IntersectionType, registerEnumType, ID } from "@nestjs/graphql";
+import { Type } from "class-transformer";
+import { IsOptional, IsString, ValidateNested, IsEnum, IsUUID } from "class-validator";
+import { OffsetBasedPaginationArgs, SortDirection } from "@comet/cms-api";
+import { TestEntity3Filter } from "./test-entity3.filter";
+import { TestEntity3Sort, TestEntity3SortField } from "./test-entity3.sort";
+
+@ArgsType()
+export class TestEntity3sArgs extends OffsetBasedPaginationArgs {
+    @Field({ nullable: true })
+    @IsOptional()
+    @IsString()
+    search?: string;
+
+    @Field(() => TestEntity3Filter, { nullable: true })
+    @ValidateNested()
+    @Type(() => TestEntity3Filter)
+    @IsOptional()
+    filter?: TestEntity3Filter;
+
+    @Field(() => [TestEntity3Sort], { nullable: true })
+    @ValidateNested({ each: true })
+    @Type(() => TestEntity3Sort)
+    sort?: TestEntity3Sort[];
```

Comment:
This is missing an `@IsOptional()` decorator.

I'm not sure if snapshot tests are _that_ good.

Reply:
oh, sorry - thanks for heads up

testing with asserting ts-morph parsed code would not make it any better. Probably even worse, because in the snapshot you noticed the missing IsOptional, with ts-morph I might not have asserted any other decorator than `@Field`. Actually running the generated code in tests is way too much effort. Imho snapshots are perfect!